### PR TITLE
Android PCMark

### PR DESCRIPTION
**Pull request by ionela-voinescu**
Add PCMark benchmark and provide an example of an android PCMark benchmark test.

### CODE REVIEW

**Comment by ionela-voinescu** [comment]
@derkling , @bjackman : I've modified these commits also according to your comments on other benchmarks.

File: libs/utils/android/workloads/pcmark.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        logcat_cmd = self._adb(
+                'logcat com.futuremark.pcmandroid.AbstractMainActivity:I *:S'\
+                .format(self._target.adb_name))
+        self._log.info("%s", logcat_cmd)
+
+        logcat = Popen(logcat_cmd, shell=True, stdout=PIPE)
+
+        # Run performance workload
+        if test_name.upper() == 'PERFORMANCE':
+            self._target.execute('input tap 750 1450')
```
Comment (derkling) [inline libs/utils/android/workloads/pcmark.py]: Here we should use relative coords... but it's a pain.

On a Pixel XL, where the screen is 2k (1440,2560), the run button has this coords:
```
   top-left:     ( 914,1630)
   bottom-right: (1340,1730)
```

File: libs/utils/android/workloads/pcmark.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+            logcat = Popen(self._adb(
+                     'logcat com.futuremark.pcmandroid.VirtualMachineState:* *:S'\
+                     .format(self._target.adb_name)),
+                     stdout=log,
+                     shell=True)
+            time.sleep(2)
+            logcat.kill()
+
+        # Parse logcat file and create db with scores
+        os.popen('grep -o "PCMA_.*_SCORE .*" {} | sed "s/ = / /g" | sort -u > {}'
```
Comment (derkling) [inline libs/utils/android/workloads/pcmark.py]: Here we should use some more portable python code, i.e. something like:
```python

SCORE_RE.compile(r'here is the regexp with the capture groups you are interested into')

results = []
with open(db_file, 'r') as fh:
    for line in fh:
        match = SCORE_RE.match(line)
        if not match:
            continue
        score = int(match.group('score'))
        results.append(score)
results.sort()
```

**Comment by valschneider** [comment]
@ionela-voinescu Do you mind if we close this PR ? If I'm missing some functionality in #475 I'll implement it free of charge.

**Pull request closed without merging**